docs/admin_guide.md

```diff
@@ -659,39 +672,6 @@ Restart (or reload) NGINX:
 sudo systemctl restart nginx
 ```
 
-#### Let's Encrypt (TLS)
```

Comment:
Restore this section. Again Certbot shouldn't be installed from APT. The package might be too old.